RELEASE_NOTES.md

```diff
@@ -2,11 +2,401 @@ _This file holds "in progress" release notes for the current release under devel
 
 # Unreleased
 
-# Chef Client Release Notes 14.TBC:
+# Chef Client Release Notes 14.0:
 
-## New Windows PowerShell mixin `powershell_exec`
+## New Resources
 
-Since our supported Windows platforms can all run .NET Framework 4.0 and PowerShell 4.0 we have taken time to add a new interop that will allow faster and safer interactions with the system PowerShell.  You will be able to use the `powershell_exec` mixin in most places where you would have previously used `powershell_out`.  For comparison, a basic benchmark test to return the `$PSVersionTable` 100 times completed 7.3X faster compared to the `powershell_out` method.  The majority of the time difference is because of less time spent in invocation.  So we believe it has big future potential where multiple calls to PowerShell are required inside (for example) a custom resource.
+Chef 14 includes a large number of resources ported from community cookbooks. These resources have been tested, improved, and functionality expanded. With these new resources in the chef-client itself the need for external cookbook dependencies and dependency management has been greatly reduced.
```

Comment:
`Chef Client` not `chef-client` (we only use the latter in copy when referring literally to the binary, not the product)

Reply:
"have had their functionality expanded" or something like that. Also there should be a comma after "itself" in the second sentence.

Reply:
Fixed and Fixed